scout/utils/scout_requests.py

```diff
@@ -322,35 +322,30 @@ def fetch_hgnc():
     return hgnc_lines
 
 
-def fetch_exac_constraint():
-    """Fetch the file with exac constraint scores
+def fetch_constraint():
```

Comment:
```suggestion
def fetch_gnomad_constraint():
```

no?

Reply:
Agree, correct, but if it had been named only "fetch_constraint" from the start I would not have had to rename it.. 😁